src/main/resources/templates/recall-summary.html

```diff
@@ -687,7 +688,14 @@
   }
 
   .c7 {
-    font-size: 10pt
+    font-size: 10pt;
+    width: 100%;
+  }
+
+  pre {
+    font-family: "Arial";
+    font-size: 10pt;
+    white-space: pre-wrap;
```

Comment:
I guess this may well all be superseded ... but I wonder why we need `white-space: pre-wrap` some places and `white-space: pre-line` in others? 🤷‍♂️